sample/src/main/java/com/pixplicity/htmlcompat/sample/MainActivity.java

```diff
@@ -110,7 +114,7 @@ public Object onSpanCreated(String tag, Object span) {
             };
             fromHtml = HtmlCompat.fromHtml(this, source,
                     HtmlCompat.FROM_HTML_SEPARATOR_LINE_BREAK_LIST_ITEM,
-                    imageGetter, tagHandler, spanCallback);
+                    imageGetter, tagHandler, spanCallback, "span");
```

Comment:
example of how overriding a tag looks.
it could be `imageGetter, tagHandler, spanCallback, "span", "em", "h2");` etc